### PR TITLE
add support for grid lists

### PR DESCRIPTION
**Pull request by PalmerAL**
Currently, this only works with two-line grid lists. I'll add support for one-line lists in a future PR.

### CODE REVIEW

File: css/material.css [hunk 1]
```diff
@@ -886,10 +886,37 @@ code {
...
+.grid-list > .tile > .tile-footer {
+	padding: 0.5em;
+}
+
+.grid-list > .tile > .tile-footer > .item-text {
+	padding: 0.5em;
+}
+
+.grid-list > .tile > .tile-footer .item-action {
+  font-size: 1.5em;
```
Comment (nt1m) [inline css/material.css]: Nit : wrong indentation

File: index.html [hunk 2]
```diff
@@ -1890,6 +1890,86 @@ <h1 class="dialog-title">Hello world</h1>
...
+					I love your Material design framework !
+				</span>
+			</span>
+			<i class="icon-message item-action"></i>
+			</div>
+		</div>
+
+		<div class="tile">
+			<img class="tile-background" src="demo-files/avatar.jpg"/>
+			<div class="tile-footer">
```
Comment (nt1m) [inline index.html:1924]: Nit : indent the tile-footer children properly please

File: index.html [hunk 3]
```diff
@@ -1890,6 +1890,86 @@ <h1 class="dialog-title">Hello world</h1>
 	&lt;/li&gt;
 &lt;/ul&gt;
 	</code>
+	
+	<div class="divider"></div>
+	<h1>Grid Lists</h1>
+	
+	<div class="grid-list">
+		<div class="tile">
+			<img class="tile-background" src="demo-files/avatar.jpg"/>
```
Comment (nt1m) [inline index.html]: Can you use tile-image instead of tile-background ? It's more generic

**Comment by nt1m** [comment]
Thanks for the PR ! I have added a few comments (mainly indentation nits). The most important one is to use tile-image instead of tile-background

**Comment by PalmerAL** [comment]
Done.

**Pull request merged**